src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -143,24 +180,24 @@ private void mapFormDateToCcdDate(String ocrFieldName, String ccdFieldName,
     private void mapAuthorisationSignedToYesOrNo(String ocrFieldName, String ccdFieldName,
                                                  List<OcrDataField> ocrDataFields, Map<String, Object> formSpecificMap) {
         ocrDataFields.stream()
-                .filter(ocrDataField -> ocrDataField.getName().equals(ocrFieldName))
-                .map(OcrDataField::getValue)
-                .findFirst()
-                .ifPresent(ocrValue -> {
-                    String ccdValue = ocrValue.trim().isEmpty() ? NO_VALUE : YES_VALUE;
-                    formSpecificMap.put(ccdFieldName, ccdValue);
-                });
+            .filter(ocrDataField -> ocrDataField.getName().equals(ocrFieldName))
+            .map(OcrDataField::getValue)
+            .findFirst()
+            .ifPresent(ocrValue -> {
+                String ccdValue = ocrValue.trim().isEmpty() ? NO_VALUE : YES_VALUE;
```

Comment:
are you sure it's never null?

Reply:
I haven't touched this. It's only styling changes.